content/java/concepts/strings/terms/subSequence/subSequence.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.subSequence()'
+Description: 'Returns a character sequence from a string'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+**`.subSquence()`** is a built-in method that returns a new character sequence that is a subsequence of the user-specified string.
+
+## Syntax
+
+```pseudo
+str.subSequence(int startIndex, int endIndex)
+```
+The `subSequence()` method will return a subsequence of the `str` string that starts with the character at the position `startIndex` and ends with the character at the position `endIndex -1`
```

Comment:
```suggestion

The `subSequence()` method will return a subsequence of the `str` string that starts with the character at the position `startIndex` and ends with the character at the position `endIndex -1`
```

Reply:
Added a line break.

Reply:
Thanks again for helpin me. Uhm, I don't really get what's wrong bout my code.

Reply:
@rubenm25 there needs to be one line break before and after each code block separating it from the other content.

Reply:
Okay! Thanks! I'll fix it now.

Reply:
I just added the necessary line breaks.